internal/service/advancedclustertpf/model_ClusterDescription20240805.go

```diff
@@ -120,7 +120,8 @@ func NewReplicationSpecsObjType(ctx context.Context, input *[]admin.ReplicationS
 
 func NewTagsObjType(ctx context.Context, input *[]admin.ResourceTag, diags *diag.Diagnostics) types.Set {
 	if input == nil {
-		return types.SetNull(TagsObjType)
+		// API Response not consistent, using empty array instead of null
```

Comment:
can you elaborate the inconsistencies here?

Reply:
Updated. Please reopen if it is still unclear